### PR TITLE
TNO-1066: New image form

### PR DESCRIPTION
**Pull request by jdtoombs**
I had misinterpreted Bobbi's mockup to have a drag and drop box; however, it was just a placeholder box. Talked to Bobbi about it and she wants to keep it and likes it!

I would also like to display the preview of the media before the snippet is saved; however, will make a separate ticket for that as it is out of scope. 

**_Also a lot of form rearrangement resulting in a lot of "green" in some of the content forms due to change in indentation._** 

- updates to upload component to match mockup
- drag and drop now enabled for both images and videos, or can click button
- new fields added to image form 
new-image-form

<hr />

Also included TNO-1142,1131:
- Fixed rounded edges on login right-panel
- Renamed CREATE NEW section to CREATE SNIPPET (as per request)
<c>image</c>

### CODE REVIEW

File: app/editor/src/components/upload/Upload.tsx [hunk 1]
```diff
@@ -37,88 +39,93 @@ export const Upload: React.FC<IUploadProps> = ({
...
-
+  const { values, setFieldValue } = useFormikContext<IContentForm>();
   const [file, setFile] = React.useState<IFile>();
-
-  /** Duration / metadata WIP */
-  // const [duration, setDuration] = useState(0);
-  const reader = new FileReader();
   const fileName = generateName(file) ?? generateName(initFile);
-
+  console.error(contentType === ContentTypeName.Snippet, 'contentType');
```
Comment (Fosol) [inline app/editor/src/components/upload/Upload.tsx:50]: Remove

File: app/editor/src/features/content/tool-bar/sections/filter/CreateNewSection.tsx [hunk 2]
```diff
@@ -45,7 +45,7 @@ export const CreateNewSection: React.FC = () => {
           </Row>
         </Col>
       }
-      label="CREATE NEW"
+      label="CREATE SNIPPET"
```
Comment (Fosol) [inline app/editor/src/features/content/tool-bar/sections/filter/CreateNewSection.tsx:48]: A snippet is used to express a Radio/TV content type.  I'm not certain it makes sense to call all new content a snippet.  Was this Bobbi's suggestion?

**Review by Fosol — CHANGES_REQUESTED** [review]

File: app/editor/src/features/content/tool-bar/sections/filter/CreateNewSection.tsx (hunk 2 above)
Comment (jdtoombs) [reply app/editor/src/features/content/tool-bar/sections/filter/CreateNewSection.tsx:48]: Yeah this was after talking to her, but I can remove for now and maybe get some clarification from Bobbi and Carolynn?

File: app/editor/src/components/upload/Upload.tsx (hunk 1 above)
Comment (jdtoombs) [reply app/editor/src/components/upload/Upload.tsx:50]: Ugh, I did a ctrl + f for console.log lol

File: app/editor/src/features/content/tool-bar/sections/filter/CreateNewSection.tsx (hunk 2 above)
Comment (jdtoombs) [reply app/editor/src/features/content/tool-bar/sections/filter/CreateNewSection.tsx:48]: Currently chatting with Bobbi - will update you when I hear back

Comment (jdtoombs) [reply app/editor/src/features/content/tool-bar/sections/filter/CreateNewSection.tsx:48]: Response: "They don't use snippet just for radio and tv, it's any content"

**Review by Fosol — APPROVED** [review]

**Pull request merged**